src/types.ts

```diff
@@ -0,0 +1,17 @@
+import type { ApiClient } from '@content-island/api-client';
+
+// Nuxt plugin augmentation to add $contentIsland to the Nuxt app
+declare module '#app' {
```

Comment:
This is the recommendation for Nuxt Plugins: https://nuxt.com/docs/guide/directory-structure/plugins#typing-plugins

Reply:
Hi @paulmelero, I don't get the necessity of doing this nor the `build.config.ts` changes since the current configuration and the `nuxt-module-build` package is generating the Typescript d.ts files that we need automatically.